sqlalchemy_aio/trio.py

```diff
@@ -11,11 +11,14 @@
 
 _STOP = object()
 
+# trio.hazmat was renamed trio.lowlevel in version 0.15.0.
+trio_version = [int(part) for part in trio.__version__.split('.')]
+trio_lowlevel = trio.hazmat if trio_version < [0, 15, 0] else trio.lowlevel
```

Comment:
There are [PEP 440](https://www.python.org/dev/peps/pep-0440/) version strings which would fail here.
What about only importing the old name if the new import fails?:

```python
try:
    import trio.lowlevel as trio_lowlevel
except ImportError
    import trio.hazmat as trio_lowlevel
```

Reply:
That sounds good. I'll switch to that.

Reply:
Unfortunately you need to use `except AttributeError` (or just `Exception`) because of the way trio exposes its modules.